Cargo.toml

```diff
@@ -106,6 +106,10 @@ reqwest = { version = "0.11", default-features = false, features = [
 ] }
 tokio-tungstenite = { version = "0.20", features = ["rustls-tls-webpki-roots"] }
 
+### DATETIME
+chrono = "0.4.26"
+chrono_locale = { git = "https://github.com/0x5eal/chrono-locale.git", tag = "v0.1.2" }
```

Comment:
Ah yes, is it the inability to publish a binary to crates.io with git dependencies?

What's blocking me from publishing it to crates.io is the crate name that is already taken, mind suggesting a couple names?

Reply:
Yep, it's because it would block us from publishing to crates.io. You could do a derivate of the word `locale`, an additional suffix, etc. it's not something our users will see anyway. Whatever you think fits best for your own fork

Reply:
Alrighty, will do!